src/styles/style.scss

```diff
@@ -0,0 +1,128 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+:root {
+  --min-margin-side: 16px;
+  --color-button: #00acdc;
+}
+
+.card {
```

Comment:
You don't need `.card` and `.stars` selectors here if you used separate files.
Let's try to fix it.
Just remove all `.card` and `.stars` styles here and import it from your files:
```
@import "./card.scss";
@import "./stars.scss";
```